src/model/bioprotocol.v1.yaml

```diff
@@ -0,0 +1,21 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: BioProtocol
+type: array
+items:
+    type: object
+    properties:
+        sequencing_number:
```

Comment:
Using camelCase elsewhere. Since this is a section ID, call it `sectionId`?

Reply:
done, is now camelCase

Reply:
renamed sequencingNumber to sectionId